search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -300,3 +322,11 @@
   (GET "/tiles"
     {params :params ctx :request-context}
     (find-tiles ctx params)))
+
+(def data-json-routes
+  "config-enabled route for data.json."
```

Comment:
Add a sentence here explaining why we need to do this (Socrata does not support hitting an endpoint that ends with collections.opendata - it must end with data.json). You might have to add ... seriously to the end of that sentence.